src/dir.proj

```diff
@@ -0,0 +1,292 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <PropertyGroup>
+    <BuildTargets>
+      BuildProjectsForNuGetPackages;
+      PublishSharedFrameworkAndSharedHost;
```

Comment:
Looks like most of this is about the sharedframework why not put it under the sharedframework path.